views/partials/_navigation.njk

```diff
@@ -1,16 +1,9 @@
 <nav class="app-c-navigation govuk-h-clearfix">
   <ul class="app-c-navigation__list">
-    <li{% if path == '' %}  class="app-c-navigation--current-page"{% endif %}>
-      <a href="/">About</a>
-    </li>
-    <li{% if 'styles' in path %}  class="app-c-navigation--current-page"{% endif %}>
-      <a href="/styles">Styles</a>
-    </li>
-    <li{% if 'components' in path %}  class="app-c-navigation--current-page"{% endif %}>
-      <a href="/components">Components</a>
-    </li>
-    <li{% if 'patterns' in path %}  class="app-c-navigation--current-page"{% endif %}>
-      <a href="/patterns">Patterns</a>
+    {% for item in navigation %}
+    <li{% if path == item.url %}  class="app-c-navigation--current-page"{% endif %}>
```

Comment:
I think this would only apply the current-page class to the 'root' of each section, and not on their children?

Also, double space. 👾

Reply:
yes, this only applies `current` for the first-level navigation, updated and commented above for the second-level navigation